src/web/theme/ui/atoms/Colors/Colors.scss

```diff
@@ -21,11 +21,11 @@ $warning: #fd7737;
 $error: #ff0000;
 
 $backgroundColor: $white;
-$fontColor: #333333;
+$fontPrimaryColor: $brandPrimary;
+$fontSecondaryColor: $brandSecondary;
 $fontDisabled: $shade05;
-$fontSecondaryColor: $shade05;
 
-$reverseBackgroundColor: $fontColor;
+$reverseBackgroundColor: $fontSecondaryColor;
```

Comment:
There is not much sense to use the `$fontSecondaryColor` variable here, you can use the brand one directly.

Reply:
Actually there is a sense :) , because the color of the background depend of your text color for readability.